debian/rules

```diff
@@ -23,6 +24,21 @@ else
 endif
 endif
 
+override_dh_gencontrol:
+ifeq ($(DIST_RELEASE),trusty)
+	echo "extra:Depends=libapt-pkg4.12 (>= 1.0.1ubuntu2.23)" >> debian/ubuntu-advantage-tools.substvars
+else ($(DIST_RELEASE),xenial))
+	echo "extra:Depends=libapt-pkg5.0 (>= 1.2.30)" >> debian/ubuntu-advantage-tools.substvars
+else ($(DIST_RELEASE),cosmic))
+	echo "extra:Depends=libapt-pkg5.0 (>= 1.7.3)" >> debian/ubuntu-advantage-tools.substvars
```

Comment:
Maybe 1.7.4? https://launchpad.net/ubuntu/+source/apt/1.7.4 same fix as above for xenial (auth.conf.d)

Reply:
done will use -updates version on each series since we  know it carries the fix (and we can test it for compatibility)